src/client.toit

```diff
@@ -148,16 +157,19 @@ class Client:
     before returning.
   */
   subscribe topic_filters/List:
```

Comment:
Hard to change now, but this API is extremely bad.
Usually you would just want to `client.subscribe "topic"`, but that doesn't work now.